BUILDING.md

```diff
@@ -1,34 +1,106 @@
-# How to build a medley release
+# How to Build Medley and make a Release
 
-Originally done only with shell scripts:
-```
-./scripts/loadup-all.sh 
-```
-to make the loadups
+# Using GitHub actions
+
+In the GitHub medley repository (Interlisp/medley) go to the Actions tab.
+
+It will list the available GitHub actions.
+Select: **Build/Push Release & Docker**. 
+
+In the middle of the screen there is a box labeled "Workflow Runs".
+There should be a row in it that states 'This workflow has a workflow_dispatch event trigger' with a drop down menu (it really looks more like a button) on the right side labeled 'Run workflow'.  Select that and you'll get a form allowing you to select the branch (I've only used Master) and enter the release name.  Enter a name or leave it empty and press the green 'Run workflow' button. The workflow should queue up and run.  
+
+Build/Push Release & Docker first builds Maiko and Medley, pushes a Medley release to the Interlisp/medley repo Releases, then makes a Docker image.
+
+The files in .github/workflows/ contain the details.
+<!-- 
+The workflow pulls the latest Maiko image from Docker Hub and the Release Assets from the latest Medley release, generally defined as medley-YYMMDD.  The Medley Docker image adds in Tight VNC Server and retrieves the two tarballs associated with a release, one containing the sysouts and the other the other needed files source, fonts, etc.  The contents are uncompressed and loaded into the Medley directory structure.
+-->
+
+# Building Medley for yourself
+
+The actual building itself is done with various shell scripts, found in the `scripts` directory in the medley repository. Most of the scripts have a minimal sanity check that they are being run from the medley repository.
+
+Historically, building the medley image (called a "sysout") was called "doing a loadup".  Back in the day, a loadup took the better part of a day, and no one would do the whole thing -- there was no automation.
+
+## Prerequisites
```

Comment:
It's worth making explicit a number of implicit assumptions or details:

- Maiko (and `ldeinit`) must be built before building Medley
- to make everything, the NoteCards source tree must be accessible to the build process
- source tree layout, e.g.
```
src
  maiko
  medley
  notecards
```
- from which directory to run which script
- whether to set `MEDLEYDIR` and to what value (`MEDLEYDIR` seems required but I'm not sure about `MAIKODIR`)

Reply:
I added some things to help address @pamoroso points above. I'm sure more is needed, but could we please get this out?